assets/vendor/react-dom.development.js

```diff
@@ -19848,4 +19848,4 @@ var reactDom = ReactDOM$3.default || ReactDOM$3;
 
 return reactDom;
 
-})));
+})));
```

Comment:
Why would this file have changed?

Reply:
I was trying to debug errors and needed to load a particular version of `react-dom`… looks like I must've had it open when restoring and my editor added a newline EOF. I'll revert it, thanks for catching that (the woes of `git add .` 😓)